utils/sparse.py

```diff
@@ -34,20 +50,31 @@ def __init__(self, model, recipe, checkpoint_recipe=None, checkpoint_epoch=float
         if self.checkpoint_recipe_manager:
             _LOGGER.info("Applying structure from checkpoint recipe")
             self.checkpoint_recipe_manager.apply_structure(
-            module=model,
-            epoch=checkpoint_epoch,
+                module=model,
+                epoch=checkpoint_epoch,
         )
 
     def state_dict(self):
+        """
+        :return: A dict object with compsed recipe
+        """
         return {
             'recipe': str(self.compose_recipes()) if self.enabled else None,
         }
 
     def apply(self):
+        """
+        Apply training recipe to model
+        """
         if self.enabled:
             self.manager.apply(self.model)
 
-    def initialize(self, start_epoch):
+    def initialize(self, start_epoch: Union[int, float]):
```

Comment:
nit add return type and `:return:` field in docstrings

Reply:
Empty return should not be added in docstrings, it adds unnecessary noise
```
style guide says this about the Returns section:

If the function only returns None, this section is not required
```